docs/PullSecret.md

```diff
@@ -14,4 +14,6 @@ oc edit secrets pull-secret -n openshift-config
 Changing the `.dockerconfigjson` in the secret and saving it will result in the MCO syncing the new pull secret with the templates which then
 result in the new pull secret being laid down on nodes.
 
-Note that the `.dockerconfigjson` field is a base64 encoded JSON. 
\ No newline at end of file
+Note that the `.dockerconfigjson` field is a base64 encoded JSON.
+
+The pull secret in `openshift-config` is a cluster wide pull secret. Changing it will cause every node in every pool, including the `master` pool to reboot.
```

Comment:
```suggestion
The pull secret in `openshift-config` is rendered into a `MachineConfig` object by the MCO which applies to all pools.  Therefore, like all `MachineConfig` changes, it will currently cause a drain and reboot for all affected machines.
```